README.md

```diff
@@ -1,4 +1,4 @@
-# Duke project template
+# duke.Duke project template
```

Comment:
I think you made the same mistake as me by using the refactor button in IntelliJ which also changed name in the README file, you might want to consider changing it back.